niceday-api/api/openapi.yaml

```diff
@@ -31,14 +31,53 @@ paths:
         - in: path
           name: userId
           schema:
-            type: integer
+            type: string
           required: true
-          description: Integer ID of the user whose data is desired
-          example: 38527
+          description: ID of the user whose data is desired
+          example: '38527'
```

Comment:
Why is it better for the userid to be a string? (sorry if I missed some discussion on this)